vm/src/pyobject.rs

```diff
@@ -472,9 +475,9 @@ impl PyObject<dyn PyObjectPayload> {
         self: PyRc<Self>,
```

Comment:
Can this be changed to take `self: PyObjectRc<Self>` instead?

Reply:
Nope -- blocked on the arbitrary_self_types feature